### PR TITLE
#2376 - Edit Program Banner Message Change

### PR DESCRIPTION
**Pull request by guru-aot**
- [x]  Adjust the 'Edit Program' banner message title to read "Program Details No Longer Editable"

### CODE REVIEW

**Review by sh16011993 — APPROVED** [review]
👍

File: sources/packages/web/src/views/institution/locations/programs/LocationProgramAddEdit.vue [hunk 1]
```diff
@@ -11,7 +11,7 @@
       <banner
         v-if="programData.hasOfferings"
         :type="BannerTypes.Success"
-        header="Students have applied financial aid for this program"
+        header="Program details no longer editable"
```
Comment (sh16011993) [inline sources/packages/web/src/views/institution/locations/programs/LocationProgramAddEdit.vue:14]: Is it supposed to be Camelcase? The PR description has it Camelcase.

Comment (andrewsignori-aot) [reply sources/packages/web/src/views/institution/locations/programs/LocationProgramAddEdit.vue:14]: The ticket itself is "Pascal case" but it is not what the UI follows.
@guru-aot can we please have the ticket updated to represent the way that you used it here?
@JasonCTang FYI

**Review by andrewsignori-aot — APPROVED** [review]
👍

**Review by ann-aot — APPROVED** [review]
👍

**Review by dheepak-aot — APPROVED** [review]
Looks Good 👍

**Comment by HRAGANBC** [comment]
Reviewed by SABC Business on Oct 10 - Signed off and looks great.

**Pull request merged**